ckan/config/middleware/flask_app.py

```diff
@@ -126,6 +126,29 @@ def _ungettext_alias():
     return dict(ungettext=ungettext)
 
 
+class BeakerSessionInterface(SessionInterface):
+    def open_session(self, app: Any, request: Any):
+        if 'beaker.session' in request.environ:
+            return request.environ['beaker.session']
+
+    def save_session(self, app: Any, session: Any, response: Any):
+        session.save()
+
+    def is_null_session(self, obj: Dict[str, Any]) -> bool:
+
+        is_null = super(BeakerSessionInterface, self).is_null_session(obj)
+
+        if not is_null:
+            # Beaker always adds these keys on each request, so if these are
+            # the only keys present we assume it's an empty session
+            is_null = (
+                sorted(obj.keys()) == [
+                    "_accessed_time", "_creation_time", "_domain", "_path"]
```

Comment:
This should also handle the possibility of setting `save_accessed_time` to false (not storing the accessed time).

Reply:
@ThrawnCA the keys are always added to the session object, regardless of the `save_accessed_time` setting: 

https://github.com/bbangert/beaker/blob/d4ab7d37387feddd0d544792d9e32533e39ff868/beaker/session.py#L222-L224

I double checked manually, so I think this works as intended